india_compliance/gst_india/doctype/isd_distribution_invoice/test_isd_distribution_invoice.py

```
@@ -0,0 +1,973 @@
+    cls.pi = make_isd_pi(cls.isd_address.name)
+    cls.branch_company = make_branch_company()
+    cls.branch_customer, cls.branch_address = make_internal_customer()
+
+
+def teardown_isd_fixtures():
+    for doctype, name in (("Customer", BRANCH_CUSTOMER), ("Company", BRANCH_COMPANY)):
+        if frappe.db.exists(doctype, name):
+            try:
+                frappe.delete_doc(doctype, name, force=True)
+            except Exception:
+                pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)